packages/lib/src/index.ts

```diff
@@ -11,7 +13,7 @@
  */
 export function createDataProp(name: string) {
   // ts:emit throws an error if this is not typed. Usually this can be inferred.
-  const prefix: string = 'data-leafygreen-ui';
+  const prefix = 'data-leafygreen-ui' as string;
```

Comment:
This change was just to make my editor stop auto-"fixing" it. There's probably a better way...

Reply:
If you're setting a `const` to a literal, I don't think you need to type it because it's then inferred.

Reply:
Btw, the inferred type of this `const` is not actually `string`. It's the string literal type `'data-leafygreen-ui'`. If it was just `string`, TS wouldn't have any unexported type to worry about.